serverless/serverless
# Use latest dotnet runtime in AWS Lambda

## What did you implement:

I've update the `csharp` and `fsharp` templates to use `netcoreapp2.1`.

## How did you implement it:

Updated the various proj files, build scripts, and yamls.

## How can we verify it:

Create lambdas from the templates, deploy them, and verify then run with no errors.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
